### PR TITLE
get bill link/url (#14)

### PR DESCRIPTION
**Pull request by kouglas**
add 'identifier', 'link', and 'url' properties to Bill type
create const baseBillUrl
return values of 'identifier', 'lin'k, and 'url'

### CODE REVIEW

File: scraper/api/bunkum.ts [hunk 1]
```diff
@@ -149,17 +155,18 @@ async function getChicagoBills() {
...
       source_id: "", // todo
       classification,
+      identifier: bill.identifier,
+      link:`${baseBillUrl + bill.identifier}`, 
+      url: `${baseBillUrl + bill.identifier}`,
     } as CiviLegislationData;
   });
 
   return results;
+  // console.log(results)
```
Comment (sartaj) [inline scraper/api/bunkum.ts:167]: Lets remove this

**Review by sartaj — COMMENTED** [review]
This looks good! Could you give an example of a one of the urls?

File: scraper/api/bunkum.ts (hunk 1 above)
Comment (kouglas) [reply scraper/api/bunkum.ts]: i just removed it. here is an example of the output which includes values for 'link', and 'url':

    "link": "https://chicago.councilmatic.org/legislation/R2023-0004224",
    "url": "https://chicago.councilmatic.org/legislation/R2023-0004224"

**Comment by sartaj** [comment]
LGTM

**Pull request merged**